### PR TITLE
update: website sitemap

### LINKED ISSUE #457: update sitemap (easy)
Work from master branch and help by updating the sitemap

### PR DESCRIPTION
**Pull request by haroon10725**
Fixes #457

### CODE REVIEW

**Comment by haroon10725** [comment]
@pikurasa Can you please review it ?

File: sitemap.txt [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-https://www.sugarlabs.org/school-administrators
-https://www.sugarlabs.org/sugar-lesson-plans
-https://www.sugarlabs.org/sugar-stories
-https://www.sugarlabs.org/sugar-for-raspberry-pi
-https://www.sugarlabs.org/sugar-for-raspbian
-https://www.sugarlabs.org/music-blocks
-https://www.sugarlabs.org/turtle-blocks-js
-https://www.sugarlabs.org/profiles
-{% for post in site.posts %}https://www.sugarlabs.org{{ post.url }}
-{% endfor %}
```
Comment (chimosky) [inline sitemap.txt:22]: Is this line no longer needed?

Comment (haroon10725) [reply sitemap.txt:22]: Sorry, it got deleted by mistake.

Comment (haroon10725) [reply sitemap.txt]: Updated.

**Review by pikurasa — APPROVED** [review]

**Comment by pikurasa** [comment]
Looks good. I'm going to merge it, but I wonder if there is an automated way to do this.

I suspect that there is.

**Comment by haroon10725** [comment]
@pikurasa I will try to find an automated way of it, I think it can be done using web scraping.

**Pull request merged**